generator/src/test/java/com/scottlogic/deg/generator/cucumber/utils/CucumberProfileSchemaValidator.java

```diff
@@ -0,0 +1,25 @@
+package com.scottlogic.deg.generator.cucumber.utils;
+
+import com.scottlogic.deg.schemas.common.ValidationResult;
+import com.scottlogic.deg.schemas.v0_1.ProfileSchemaValidator;
+
+import java.io.File;
+import java.io.InputStream;
+import java.util.ArrayList;
+
+public class CucumberProfileSchemaValidator implements ProfileSchemaValidator {
```

Comment:
Note: In future this could move to the main code-base called `NullProfileSchemaValidator` to solve #782

Reply:
when we make schema validation optional, we will not call a validtor at all, rather than call a null validator